var/spack/repos/builtin/packages/bart/package.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Bart(MakefilePackage, CudaPackage):
+    """BART: Toolbox for Computational Magnetic Resonance Imaging"""
+
+    homepage = "https://mrirecon.github.io/bart/"
+    url      = "https://github.com/mrirecon/bart/archive/v0.5.00.tar.gz"
+
+    version('0.5.00', sha256='30eedcda0f0ef3808157542e0d67df5be49ee41e4f41487af5c850632788f643')
+
+    # patch to fix build with MKL
+    patch('https://github.com/mrirecon/bart/commit/b62ca4972d5ac41a44217a5c27123c15daae74db.patch',
+          sha256='8fd1be181da928448da750b32d45ee6dce7ba6af0424617c4f8d653cf3f05445',
+          when='@0.5.00')
+
+    # patch to fix Makefile for openblas and cuda
+    patch('Makefile.patch')
+
+    # patch to set path to bart
+    patch('bart_path.patch')
+
+    depends_on('libpng')
+    depends_on('fftw')
+    depends_on('blas')
+    depends_on('lapack')
+    depends_on('py-numpy')
+    depends_on('py-matplotlib')
+    extends('python')
+
+    conflicts('^atlas', msg='BART does not currently support atlas')
+
+    def edit(self, spec, prefix):
+        env['PREFIX'] = prefix
+        env['FFTW_BASE'] = spec['fftw'].prefix
+
+        if spec['blas'].name in 'openblas':
+            env['OPENBLAS'] = '1'
+
+        if spec['blas'].name in ['intel-mkl', 'intel-parallel-studio']:
```

Comment:
```suggestion
        elif spec['blas'].name in ['intel-mkl', 'intel-parallel-studio']:
```

Reply:
That would not be correct here as `env[BLAS_BASE']`, from the `else` clause needs to also apply to the openblas case. It is only when 'MKL' is present that BLAS_BASE does not get set.